src/Makefile

```diff
@@ -0,0 +1,44 @@
+MAINFILE = main
+LIBFILE = blink
+PORT = /dev/ttyUSB0
+DEVICE = atmega328p
+FRECUENCY = 16000000UL
+PROGRAMMER = arduino
+BAUD = 57600
+COMPILER = avr-gcc
+UPLOADER = avrdude
+
+LINT_TOOL = cpplint
+LINT_FLAGS = --recursive --quiet
+
+
+dockeride:
+ifndef YOUARECONTAINERIZEDBUDDY
+	../tools/dockeride -p $(PWD)/..
+else
+	echo "Alredy dockerized, pal!"
+endif
+
+	$(info )
```

Comment:
:writing_hand: change to `$(info Welcome to the docker image for programming AVR microcontrollers.)`